crates/nu-command/src/system/exec.rs

```diff
@@ -62,30 +57,22 @@ fn exec(
     stack: &mut Stack,
     call: &Call,
 ) -> Result<PipelineData, ShellError> {
+    let config = engine_state.get_config();
     let name: Spanned<String> = call.req(engine_state, stack, 0)?;
     let name_span = name.span;
-
-    let redirect_stdout = call.has_flag("redirect-stdout");
-    let redirect_stderr = call.has_flag("redirect-stderr");
-    let redirect_combine = call.has_flag("redirect-combine");
-    let trim_end_newline = call.has_flag("trim-end-newline");
-
-    let external_command = create_external_command(
-        engine_state,
-        stack,
-        call,
-        redirect_stdout,
-        redirect_stderr,
-        redirect_combine,
-        trim_end_newline,
-    )?;
-
-    let cwd = current_dir(engine_state, stack)?;
-    let mut command = external_command.spawn_simple_command(&cwd.to_string_lossy())?;
-    command.current_dir(cwd);
-    command.envs(&external_command.env_vars);
```

Comment:
Just tested the new code, and new env variables are not applied since this line is gone.
E.g.:
```nushell
$env.TEST = 1
exec printenv
# Does not show: TEST=1
```

Reply:
For the same reason, `PWD` is not updated either:
```nushell
# started in home dir
cd some_dir
exec printenv
# Does not show: PWD=/home/user/some_dir
```

Reply:
Ya, I hesitated on removing that part. I'll have to look into it a bit. I'm not sure it can be done exactly like that now. Thanks for the tip. You should feel free to make changes to this PR if you find other things that don't look right.

Reply:
There is an execvpe which allows the environment to be passed in but I can't seem to get it to work. 😭

Reply:
Yeah, I just reading about that as well. I might try and take a shot and see how it goes :skull:

Reply:
Windows environment blocks have the format

> *Var1*=*Value1*\0
> *Var2*=*Value2*\0
> *Var3*=*Value3*\0
> ...
> *VarN*=*ValueN*\0\0

as documented with [`GetEnvironmentStrings`](https://learn.microsoft.com/en-us/windows/win32/api/processenv/nf-processenv-getenvironmentstrings)